config/locales/en.yml

```diff
@@ -7,24 +7,21 @@ en:
   invalid_updated_since_filter: "The filter '#/updated_since' must be a valid ISO 8601 date"
   invalid_page_parameters: "The '#/page[page]' and '#/page[per_page]' parameter values must be a valid positive number"
   invalid_data_structure: correct json data structure required. See API docs for reference
-  schedule_invalid_for_course: Selected schedule is not valid for the course
 
   errors:
     email:
       invalid: Enter a valid email address
 
-  application:
-    missing_application: "The entered '#/application' is missing from your request. Check details and try again."
+  application: &application
```

Comment:
do you mind splitting these changes into another pr regarding the locales, as from the declaration pr I'm not sure all are working as expected, and prefer we have a ticket for them so we can test properly, let's chat tomorrow :)